GDAS-validation/run_job.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+# run_job.sh
+# using rocotoboot, point to a global-workflow experiment
+# and run a specified job for a specified cycle
+
+usage() {
+  set +x
+  echo
+  echo "Usage: $0 -c /path/to/config.sh -t taskname"
+  echo
+  echo "  -c  path to config shell script to source"
+  echo "  -t  name of workflow task to run"
+  echo
+  exit 1
+}
+
+
+while getopts ":c:t:h" opt; do
+  case $opt in
+    c)
+      ConfigPath=$OPTARG
+      ;;
+    t)
+      TaskName=$OPTARG
+      ;;
+  esac
+done
+
+if [ $# -ne 4 ]; then
+  echo "Incorrect number of arguments"
+  usage
+fi
+
+# first, source the config script
```

Comment:
We need to emphasize that users need to create their own `config.sh` prior to executing `run_job.sh`.   They may be looking for `config.sh` in the JEDI-T2O clone or in the EXPDIR created by `setup_workspace.sh`.

Reply:
Nothing to commit here.  This is just a comment to be remembered / captured in a user readme.